source/jobs/JobsFeature.cpp

```diff
@@ -167,10 +173,19 @@ void JobsFeature::subscribeToUpdateJobExecutionStatusAccepted(string jobId)
         std::bind(
             &JobsFeature::updateJobExecutionStatusAcceptedHandler, this, std::placeholders::_1, std::placeholders::_2),
         std::bind(&JobsFeature::ackSubscribeToUpdateJobExecutionAccepted, this, std::placeholders::_1));
+
+    if (std::future_status::timeout == updateAcceptedPromise.get_future().wait_for(std::chrono::seconds(10)))
+    {
+        ostringstream errorMessage;
+        errorMessage << "Failed to subscribe to updateJobExecutionStatusAccepted!";
```

Comment:
We should update this error message from Failed to timed out. Here the subscription call has timed out.

Reply:
I think the fact that it timed out indicates that it has failed, but I agree that it makes it a bit clearer for debugging so I will update it